src/apps/kandel/kandelParser.ts

```diff
@@ -38,6 +55,7 @@ export const KandelParserApp: AppFactory = ethApp.parseContractLogsApp({
   },
   map: {
     tx: ({ tx, block, args }) => {
+      const chainlistId = parseInt((args as Args).chainlistId!);
```

Comment:
If `chainlistId` is optional, we should define the behavior for the case it is not set.